VSMacLocator/VSMacInstance.cs

```diff
@@ -67,7 +67,7 @@ void AddInstancesFromBundleId (string bundleId, bool isPreview = false)
                 var values = MacInterop.GetStringValuesFromPlist(infoPlistPath, bundleShortVersionKey, releaseIdKey);
                 var bundleVersion = values[bundleShortVersionKey];
                 var releaseId = values[releaseIdKey];
-                var binDir = Path.Combine(bundlePath, "Contents", "Resources", "lib", "monodevelop", "bin");
+                var binDir = Path.Combine(bundlePath, "Contents", "MonoBundle");
```

Comment:
I thought about that, but wasn't sure where to draw the line...which version breaks this and needs the new path?

Reply:
Perhaps it could probe for a known file in the newer directory, fall back to probing for a known file in the older directory, and exit with an error if that fails too?

Reply:
Ok, added the logic to probe for the branding.xml file, if it's not found then return null;